Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2764,7 +2764,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
 #if USE(QUICK_LOOK)
     void didStartLoadForQuickLookDocumentInMainFrame(const String& fileName, const String& uti);
-    void didFinishLoadForQuickLookDocumentInMainFrame(ShareableResourceHandle&&);
+    void didFinishLoadForQuickLookDocumentInMainFrame(WebCore::ShareableResourceHandle&&);
```

Comment:
Nit don't see that change in WebPageProxy.cpp.

Reply:
Because it has `using namespace WebCore;`